Source/WebCore/platform/ios/VideoFullscreenInterfaceAVKit.mm

```diff
@@ -368,7 +526,12 @@ @implementation WebAVPlayerViewController {
 #endif
     RetainPtr<AVPlayerViewController> _avPlayerViewController;
     RetainPtr<NSTimer> _startPictureInPictureTimer;
-    id<AVPlayerViewControllerDelegate_WebKitOnly> _delegate;
+    WeakObjCPtr<WebAVPlayerViewControllerDelegate> _delegate;
+
+#if HAVE(PIP_CONTROLLER)
+    RetainPtr<AVPictureInPictureController> _pip;
```

Comment:
I realise now that this name wasn't the most appropriate. All other BlahController object have _blahController for name